controllers/cache_handler.go

```diff
@@ -256,6 +260,10 @@ func (h DisruptionTargetWatcherHandler) findNotifiableEvents(eventsToSend map[st
 					"message", event.Message,
 					"timestamp", event.LastTimestamp.Time.Unix(),
 				)
+			} else if event.Reason == "Killing" && strings.Contains(event.Message, "Stopping container") && eventsToSend[chaosv1beta1.EventContainerWarningState] {
+				// this event indicates a safe killing of a container (can occur when we rollout or manually delete a pod for example)
+				// we remove the warning state event if it has been created when we compared the state of the containers
+				eventsToSend[chaosv1beta1.EventContainerWarningState] = false
```

Comment:
what is the difference between marking it as `false` versus completely remove it from `eventsToSend` map?

Reply:
We can just remove it, it does not change anything. Thanks!